src/Workspaces/VisualBasic/Portable/Extensions/SyntaxTokenExtensions.vb

```diff
@@ -57,6 +57,31 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Extensions
         End Function
 
         <Extension()>
+        Public Function IsGlobalStatementContext(token As SyntaxToken, position As Integer) As Boolean
```

Comment:
This function checks if the caret is in the context of a global statement. `position` is the position of the caret and not the token.
For `Console.|` example caret is in the context of a member access. But for `Dim x = 1: |` the caret is in the context of a global statement.
Also this function has been named as its C# equivalent.